root-io/src/core/tstreamerinfo.rs

```diff
@@ -1,7 +1,10 @@
 use quote::*;
-use nom::*;
+// use nom::*;
+use nom::HexDisplay;
+use nom::{IResult, be_u16, be_u32};
 
-use ::core::*;
+
+use crate::core::{*, Context};
```

Comment:
Yeah there was an error that `Context` was ambiguous, so there must be a new one exported by nom.

"crate"-scope may be a new feature. I was initially confused importing from "core" things that were not in rust's [core](https://doc.rust-lang.org/core/index.html), so I thought I'd make it a little more explicit.
 
It looks like this particular case I screwed up, because it was `use ::core` and not `use core`,

Reply:
yeah, crate scope is new in `1.30`, invented to resolve ambiguities